dev_tools/qualtran_dev_tools/reference_docs.py

```diff
@@ -232,11 +232,29 @@ def generate_ref_docs(reporoot: Path):
     # them all here.
 
     import qualtran
-    from qualtran import cirq_interop, drawing, resource_counting
-    from qualtran.simulation import classical_sim, quimb_sim
+    from qualtran import (
+        cirq_interop,
+        drawing,
+        linalg,
+        resource_counting,
+        serialization,
+        surface_code,
+        testing,
+    )
+    from qualtran.simulation import classical_sim, tensor
 
     # prevent unused warnings:
-    assert [drawing, resource_counting, cirq_interop, quimb_sim, classical_sim]
```

Comment:
Is this better than just a pylint ignore?

Reply:
hmm this works in pycharm as well but maybe there's some secret comment that works for all systems. but the assert also works